### PR TITLE
SPCTR-1891: Fix - Forms phone country code color and select control bgActiveColor not working as intended.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Forms input field phone country code color and select control bgActiveColor not working as intended.
https://brainstormforce.atlassian.net/browse/SPCTR-1891

### Screenshots

here is the screencast: [Screencast at July 5th 2023 - 11.mp4](https://d.pr/v/CZOSLu)

### How has this been tested?

- Drop Form block
- Add Phone Block
- Go to Style > Input
- Apply colors
- The country value must have input color
- Background Active color
- Dropdown must be the same in the Front end & Editor

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -166,9 +166,14 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Forms - Resolved an issue in which phone country-code input color would not work as intended on the backend.
+* Fix: Forms - Resolved an issue in which phone country-code select-control background active color would not work as intended on the backend.
+
 ### 2.6.8 - WEDNESDAY, 21st JUNE 2023 ###
 * Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
 * Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
+* Fix: Form - Resolved an issue where the submit button is visible on the frontend when no preset from the variation picker is selected.
```
Comment (AbhijitNage123) [inline README.md:176]: @rajmohanreddy774 Please remove the duplicate changelog.

File: readme.txt [hunk 2]
```diff
@@ -166,9 +166,14 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
 == Changelog ==
 
+= x.x.x =
+* Fix: Forms - Resolved an issue in which phone country-code input color would not work as intended on the backend.
+* Fix: Forms - Resolved an issue in which phone country-code select-control background active color would not work as intended on the backend.
+
 = 2.6.8 - WEDNESDAY, 21st JUNE 2023 =
 * Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
 * Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
+* Fix: Form - Resolved an issue where the submit button is visible on the frontend when no preset from the variation picker is selected.
```
Comment (AbhijitNage123) [inline readme.txt:176]: @rajmohanreddy774 Please remove the duplicate changelog.

File: src/blocks/forms/styling.js [hunk 3]
```diff
@@ -577,7 +577,7 @@ function styling( attributes, clientId, name ) {
 		' .uagb-forms-field-set:hover .uagb-forms-input select': {
 			'color': inputplaceholderHoverColor,
 		},
-		' #inspector-select-control-0': {
+		' .components-base-control__field:hover .components-select-control__input': {
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/forms/styling.js:582]: No, removed.

File: src/blocks/forms/styling.js [hunk 4]
```diff
@@ -577,6 +577,9 @@ function styling( attributes, clientId, name ) {
 		' .uagb-forms-field-set:hover .uagb-forms-input select': {
 			'color': inputplaceholderHoverColor,
 		},
+		' #inspector-select-control-0': {
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/forms/styling.js]: Yes it is changing so removed.

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -178,6 +178,8 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Buttons - Resolved an issue in which the space between the icon and text would not apply in RTL sites.
 * Fix: Image - Resolved an issue in which the tablet and mobile border css would not be applied as intended.
 * Fix: Forms - Reworked the e-mail sending functionality to ensure that only one e-mail is sent when CC and BCC addresses are specified.
+* Fix: Forms - Resolved an issue in which phone country-code input color would not work as intended on the backend.
+* Fix: Forms - Resolved an issue in which phone country-code select-control background active color would not work as intended on the backend.
```
Comment (bsf-zanev) [inline README.md:182]: Can we make these two into 1 changelog entry?

```suggestion
* Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update needed.

**Review by adi3890 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (adi3890) [reply README.md:182]: @rajmohanreddy774 pleasse take care of this.
Forwarding to QA

Comment (rajmohanreddy774) [reply README.md]: Done.

**Pull request merged**